pkg/service/test/test.go

```diff
@@ -922,19 +927,14 @@ func (t *tester) testHttp(tc models.TestCase, actualResponse *models.HttpResp, n
 				logDiffs.PushBodyDiff(fmt.Sprint(tc.HttpResp.Body), fmt.Sprint(actualResponse.Body), bodyNoise)
 			}
 		}
-		t.mutex.Lock()
-		logger.Printf(logs)
-		err := logDiffs.Render()
```

Comment:
why exactly have you removed this render function?

Reply:
Revised